### ISSUE TITLE
Replicate Amit Gaiki's results

### ISSUE THREAD

**Issue by prasadtalasila** [prasadtalasila/BITS-Darshini#5]
Replicate the setup of Amit Gaiki and show the demonstration.

**Comment by crygnus** [comment]
I have successfully replicated Amit Gaiki's results on my machine. I will now replicate the same procedure on my AWS EC2 instance which is an Ubuntu server.

A side note: I found that deploying has a lot of manual stages in it. We should think of some way of automating it. Maybe through some simple script. So far I have found out one more issue. I will open up a separate issue in order to track it better.

**Comment by prasadtalasila** [comment]
As you mentioned a script would be a good addition. Please make sure that all the code you are using are either coming from the existing repo code base. In case anything is missing in the repo, that can be committed into the repo.

**Comment by crygnus** [comment]
I have set up the application on my remote AWS EC2 instance. The url to access the web application is as follows - 
http://ec2-52-34-103-47.us-west-2.compute.amazonaws.com:8080/protocolanalyzer

I have yet to set up Jenkins which will take care of continuous deployment. Once that is done, I will close this issue.

Note: Above url is not accessible via bits-wif (use a Lan connection). I faced similar issues while sshing into my remote hosts.

**Comment by crygnus** [comment]
Correcting the previous comment. The url accessing/sshing into hosts is not happening via bits-wifi. On lan, it can still be accessed.

**Comment by crygnus** [comment]
I have set up jenkins on the aforementioned server for continuous integration. I have configured it to build and deploy the application on Tomcat after any change is pushed to this repo's master branch. One can log into jenkin's dashboard and trigger the build as well. The url for jenkins dashboard is as follows - 
http://ec2-52-34-103-47.us-west-2.compute.amazonaws.com:9090/

Note: You need to be logged in to trigger build or change any other configurations.
